plugins/meta/tuning/tuning.go

```diff
@@ -542,3 +545,27 @@ func readAllowlist() (bool, []string, error) {
 	}
 	return true, allowList, nil
 }
+
+type sysctlKey string
+
+type sysctlCheck struct {
+	SysCtl map[sysctlKey]string `json:"sysctl"`
+}
+
+var sysctlDuplicatesMap = map[sysctlKey]interface{}{}
+
+func (d *sysctlKey) UnmarshalText(data []byte) error {
+	key := sysctlKey(string(data))
+	if _, exists := sysctlDuplicatesMap[key]; exists {
+		return errors.New("duplicated sysctl keys are not allowed")
+	}
+	sysctlDuplicatesMap[key] = ""
+	return nil
+}
+
+func validateSysctlConflictingValues(data []byte) error {
+	// Need to reset values for use in tests
+	//sysctlDuplicatesMap = map[sysctlKey]interface{}{}
```

Comment:
Did you mean to leave this commented out?

Reply:
That's taken care of in the tests. Fixed.
Thanks

Reply:
> @mmirecki @dcbw observes we should do this when parsing the config for CHECK too, right?

@squeed @dcbw Done